llvm/tools/llvm-dis/llvm-dis.cpp

```diff
@@ -82,6 +82,9 @@ static cl::opt<bool> PrintThinLTOIndexOnly(
 
 extern cl::opt<bool> WriteNewDbgInfoFormat;
 
+extern cl::opt<bool> WriteNewDbgInfoFormat;
```

Comment:
Unintentional addition?

Reply:
Whoops, yep, that just fell out of a badly applied patch. Thanks!